dracut.sh

```diff
@@ -2018,11 +2025,13 @@ if [[ $uefi = yes ]]; then
 
     [[ -s $dracutsysrootdir/usr/lib/os-release ]] && uefi_osrelease="$dracutsysrootdir/usr/lib/os-release"
     [[ -s $dracutsysrootdir/etc/os-release ]] && uefi_osrelease="$dracutsysrootdir/etc/os-release"
+    [[ -s "${dracutsysrootdir}${uefi_splash_image}" ]] || uefi_splash_image=/dev/null
 
     if objcopy \
            ${uefi_osrelease:+--add-section .osrel=$uefi_osrelease --change-section-vma .osrel=0x20000} \
            --add-section .cmdline="${uefi_outdir}/cmdline.txt" --change-section-vma .cmdline=0x30000 \
-           --add-section .linux="$kernel_image" --change-section-vma .linux=0x40000 \
+           --add-section .splash="$uefi_splash_image" --change-section-vma .splash=0x40000 \
```

Comment:
```
  ${uefi_splash_image: --add-section .splash="$uefi_splash_image" --change-section-vma .splash=0x40000}
```
no need to set it to /dev/null ... just leave it empty

Reply:
I updated the code as you suggested.